Packs/CommonScripts/Scripts/DockerHardeningCheck/DockerHardeningCheck.py

```diff
@@ -37,7 +38,7 @@ def check_memory(target_mem: str, check_type: str) -> str:
         str -- error string if failed
     """
     size = mem_size_to_bytes(target_mem)
-    if check_type == "allocation":
+    if check_type == "allocate":
```

Comment:
Can you please refer us to this change documentation?

Reply:
I believe the original `allocation` was a typo as all of the documentation refers to `allocate`. Links to docs below:

[DockerHardeningCheck Script on XSOAR Dev Site](https://xsoar.pan.dev/docs/reference/scripts/docker-hardening-check#inputs)

> The memory check type to perform: cgroup - check memory cgroup configuration, allocate - try allocating actual memory and verify that the allocation fails. Note the allocate test on some configurations may cause the container to be killed by the linux memory manager and the whole test will then time out.

[Script YML Showing Argument Options](https://github.com/demisto/content/blob/a4b9233fbfda2e5243cdc1844fe9593b11386a22/Packs/CommonScripts/Scripts/DockerHardeningCheck/DockerHardeningCheck.yml#L19)

>   predefined:
>  - cgroup
>  - allocate

[Comments for the function in code](https://github.com/demisto/content/blob/a4b9233fbfda2e5243cdc1844fe9593b11386a22/Packs/CommonScripts/Scripts/DockerHardeningCheck/DockerHardeningCheck.py#L34)

> check_type {str} -- How to check either: cgroup (check configuration of cgroup) or allocate (check actual allocation)